packages/ui/src/components/navbar/navbar.tsx

```diff
@@ -1,9 +1,9 @@
 import { useMemo } from 'react'
 import { Link, useLocation, useNavigate } from 'react-router-dom'
 
-import { Button, Icon, NavbarProjectChooser, ScrollArea, Spacer } from '@/components'
+import { Button, Icon, IThemeStore, NavbarProjectChooser, ScrollArea, Spacer } from '@/components'
```

Comment:
That bit of Hungarian notation really stands out as being different. Will open a ticket to start standardising on naming.